src/navigation/checkout/Loopeat.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react'
+import { Linking, FlatList } from 'react-native'
+import { Box, Text, Button, VStack, Pressable, HStack, Heading } from 'native-base'
+import { useNavigation } from '@react-navigation/native'
+import { connect } from 'react-redux'
+import _ from 'lodash'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { useTranslation } from 'react-i18next'
+
+import { selectCart } from '../../redux/Checkout/selectors'
+import { updateLoopeatReturns } from '../../redux/Checkout/actions'
+import { formatPrice } from '../../utils/formatting'
+
+function getNameFromId(formatId, formats) {
+  const format = formats.find(f => f.id === formatId)
+  return format.title
+}
+
+function getPriceFromId(formatId, formats) {
+  const format = _.find(formats, f => f.id === formatId)
+  return format.cost_cents
+}
+
+const getReturnsTotalAmount = (returns, formats) => returns.reduce(
+  (total, container) => total + (getPriceFromId(container.format_id, formats) * container.quantity),
+  0
+)
+
+function Loopeat({ cart, updateLoopeatReturns }) {
+
+	const navigation = useNavigation()
+	const { t } = useTranslation()
+
+	const {
+		containers,
+		containersCount,
+		requiredAmount,
+		creditsCountCents,
+		formats,
+		returns: initialReturns,
+	} = cart.loopeatContext
+
+	const [ returns, setReturns ] = useState(initialReturns)
```

Comment:
Could these two variables called "returns" trigger any error?

Reply:
No, AFAIK only `return` is a reserved keyword.

Reply:
Sorry, my fault. I've got confused with `returns` in line 40 and `returns` in line 43 but now I realise that there is no collision between them.